l10n_it_fatturapa/bindings/fatturapa.py

```diff
@@ -116,10 +116,13 @@ def CreateFromDocument(xml_string):
                     problems.append(msg)
                     _logger.warn(msg)
 
-    # fix trailing spaces in <PECDestinatario/>
+    # fix trailing spaces in <PECDestinatario/> and <Email/>
     for pec in root.xpath("//PECDestinatario"):
         pec.text = pec.text.rstrip()
 
+    for email in root.xpath("//Email"):
+        email.text = email.text.rstrip()
```

Comment:
Perchè non `strip()`?

Reply:
> Perchè non `strip()`?

Avrebbe più senso in effetti.
Non so se c'è un motivo particolare, meglio chiedere direttamente a @TheMule71

Reply:
concordo, meglio `strip()`!

Reply:
@TheMule71 opinioni?

Reply:
Visto che non ci sono state risposte vado per il "silenzio assenso." 
Aggiorno entrambi usando `strip()`.